src/aiohomeconnect/model/event.py

```diff
@@ -430,3 +432,16 @@ class EventKey(StrEnum):
     REFRIGERATION_FRIDGE_FREEZER_SETTING_SUPER_MODE_REFRIGERATOR = (
         "Refrigeration.FridgeFreezer.Setting.SuperModeRefrigerator"
     )
+
+
+class EventTypes(StrEnum):
```

Comment:
We use singular form for enum names.
```suggestion
class EventType(StrEnum):
```